DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Operations/Receive/RhinoHostObjectBuilder.cs

```diff
@@ -58,8 +58,7 @@ CancellationToken cancellationToken
   )
   {
     RhinoDoc doc = _contextStack.Current.Document;
-
-    var rootLayerIndex = doc.Layers.Find(Guid.Empty, baseLayerName, RhinoMath.UnsetIntIndex);
+    var rootLayerIndex = _contextStack.Current.Document.Layers.Find(Guid.Empty, baseLayerName, -1);
```

Comment:
Should we keep the RhinoMath.UnsetIntIndex ?

Reply:
Unsure, the original conversion set it to -1. It is somewhat more readable to use `UnsetIntIndex` but it is somewhat of a Rhino invention.

Reply:
Actually, I just looked at it again and it's cleaner with UnsetIntIndex as we can also check for equality on that value in the condition below. Doing this change.